docs/api/item.md

```diff
@@ -70,2058 +82,91 @@ The below chart details the item slots and where it will place the element insid
-<ion-list lines="inset">
-  <ion-item>
-    <ion-label>Inset Lines Item 1</ion-label>
-  </ion-item>
-
-  <ion-item>
-    <ion-label>Inset Lines Item 2</ion-label>
-  </ion-item>
-</ion-list>
-
-<!-- List No Lines -->
-<ion-list lines="none">
-  <ion-item>
-    <ion-label>No lines Item 1</ion-label>
-  </ion-item>
-
-  <ion-item>
-    <ion-label>No lines Item 2</ion-label>
-  </ion-item>
-
-  <ion-item>
-    <ion-label>No lines Item 3</ion-label>
-  </ion-item>
-</ion-list>
-```
+## Helper & Error Text
 
+Helper & error text can be used inside of an item with an input by slotting a note in the `"helper"` and `"error"` slots. The error slot will not be displayed unless the `ion-invalid` class is added to the `ion-item`. In Angular, this is done automatically through form validation. In JavaScript, React and Vue, the class needs to be manually added based on your own validation.
 
-### Media Items
-
-```html
-<ion-item button onclick="testClick()">
-  <ion-avatar slot="start">
-    <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAAAAACH5BAAAAAAALAAAAAABAAEAAAICTAEAOw==">
-  </ion-avatar>
-  <ion-label>
-    Avatar Start, Button Item
-  </ion-label>
-</ion-item>
-
-<ion-item href="#">
-  <ion-label>
-    Thumbnail End, Anchor Item
-  </ion-label>
-  <ion-thumbnail slot="end">
-    <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAAAAACH5BAAAAAAALAAAAAABAAEAAAICTAEAOw==">
-  </ion-thumbnail>
-</ion-item>
-
-<ion-item>
-  <ion-thumbnail slot="start">
-    <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAAAAACH5BAAAAAAALAAAAAABAAEAAAICTAEAOw==">
-  </ion-thumbnail>
-  <ion-label>
-    <h2>H2 Title Text</h2>
-    <p>Button on right</p>
-  </ion-label>
-  <ion-button fill="outline" slot="end">View</ion-button>
-</ion-item>
-
-<ion-item button onclick="testClick()">
-  <ion-thumbnail slot="start">
-    <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAAAAACH5BAAAAAAALAAAAAABAAEAAAICTAEAOw==">
-  </ion-thumbnail>
-  <ion-label>
-    <h3>H3 Title Text</h3>
-    <p>Icon on right</p>
-  </ion-label>
-  <ion-icon name="close-circle" slot="end"></ion-icon>
-</ion-item>
-```
+import HelperError from '@site/static/usage/item/helper-error/index.md';
 
-### Buttons in Items
-
-```html
-<ion-item>
-  <ion-button slot="start">
-    Start
-  </ion-button>
-  <ion-label>Button Start/End</ion-label>
-  <ion-button slot="end">
-    End
-  </ion-button>
-</ion-item>
-
-<ion-item>
-  <ion-button slot="start">
-    Start Icon
-    <ion-icon name="home" slot="end"></ion-icon>
-  </ion-button>
-  <ion-label>Buttons with Icons</ion-label>
-  <ion-button slot="end">
-    <ion-icon name="star" slot="end"></ion-icon>
-    End Icon
-  </ion-button>
-</ion-item>
-
-<ion-item>
-  <ion-button slot="start">
-    <ion-icon slot="icon-only" name="navigate"></ion-icon>
-  </ion-button>
-  <ion-label>Icon only Buttons</ion-label>
-  <ion-button slot="end">
-    <ion-icon slot="icon-only" name="star"></ion-icon>
-  </ion-button>
-</ion-item>
-```
+<HelperError />
```

Comment:
Is the `ion-touched` logic needed here?

Reply:
Yeah so without adding `ion-touched` to the JavaScript, Vue, and React examples the label does not change to the proper color for an `ion-invalid` state. This results in the following focused & unfocused states:

![Screen Shot 2022-09-13 at 7 18 02 PM](https://user-images.githubusercontent.com/6577830/190025834-f5b34bab-35a0-456b-b89d-722893a977fe.png)
![Screen Shot 2022-09-13 at 7 18 30 PM](https://user-images.githubusercontent.com/6577830/190025943-ef217226-d3f3-4f56-9817-40899a5660e3.png)

Angular handles these states automatically so it looks like this for both focused & unfocused states:

![Screen Shot 2022-09-13 at 7 19 23 PM](https://user-images.githubusercontent.com/6577830/190026060-e90c9255-e462-4789-a197-f9c377e092f2.png)

At first I wasn't adding `ion-touched` but I added them in after realizing this because Angular has the correct behavior. I figured we should match that for all of the other examples so users know what classes the validation / error text requires since this isn't well documented anywhere.

Also, when you first start typing in any of the examples, you'll see the error text / highlight turn red as soon as you start typing, but the label does not. This is actually a [framework bug](https://github.com/ionic-team/ionic-framework/issues/21643). The label has the correct behavior, the error text and highlight color should only display / change color after the `ion-touched` class has been added (so on blur of the input).